std_msgs/String text
sensor_msgs/RegionOfInterest roi